odata-protocol/11.5 Operations.md

```diff
@@ -423,10 +423,10 @@ default content type will be used.
 The client can request whether any results from the action be returned
 using the [`return`](#Preferencereturnrepresentationandreturnminimal) preference.
 
-Actions that create and return a single entity follow the rules for
-[entity creation](#CreateanEntity) and return a
+If an invocation of an action results in the creation and return of a single entity,
```

Comment:
This reads better, but I can't quote a grammar or style rule justifying that change:

```suggestion
If the invocation of an action results in the creation and return of a single entity,
```